modules/navier_stokes/src/userobjects/PINSFVRhieChowInterpolator.C

```diff
@@ -58,7 +59,19 @@ PINSFVRhieChowInterpolator::PINSFVRhieChowInterpolator(const InputParameters & p
   const auto porosity_name = deduceFunctorName(NS::porosity);
 
   for (const auto tid : make_range(libMesh::n_threads()))
+  {
     _epss[tid] = &UserObject::_subproblem.getFunctor<ADReal>(porosity_name, tid, name());
+
+    if (_smoothing_layers > 0)
+    {
+      if (!UserObject::_subproblem.hasFunctor(NS::smoothed_porosity, tid))
+        // Smoothed porosity is only an envelope at this point, it will be set during pinsfvSetup()
+        UserObject::_subproblem.addFunctor(NS::smoothed_porosity, _smoothed_eps, tid);
```

Comment:
At some point maybe we'll want to add a user ability to control what the name is if there's multiple, disconnected fluid flow regions, but even there they may choose to use the same variable sets anyway. So I don't think we need to do anything about this right now